micro_ros_setup/README.md

```diff
@@ -15,9 +15,11 @@ This ROS 2 package is the entry point for building micro-ROS apps for different
 
 | RTOS | Platform | Example |
 |-|-|-|
-| [Nuttx](https://nuttx.org/) | Olimex STM32-E407, STM32F4Discovery | `nuttx olimex-stm32-e407` | 
+| [Nuttx](https://nuttx.org/) | Olimex STM32-E407, STM32F4Discovery ** | `nuttx olimex-stm32-e407` | 
 | [FreeRTOS](https://www.freertos.org/) | [Crazyflie 2.1](https://www.bitcraze.io/crazyflie-2-1/) | `freertos crazyflie21` | 
-| [FreeRTOS](https://www.freertos.org/) | [Olimex STM32-E407](https://www.olimex.com/Products/ARM/ST/STM32-E407/open-source-hardware) | `freertos olimex_e407` | 
+| [FreeRTOS](https://www.freertos.org/) | [Olimex STM32-E407](https://www.olimex.com/Products/ARM/ST/STM32-E407/open-source-hardware) | `freertos olimex-stm32-e407` | 
+| [Zephyr](https://www.zephyrproject.org/) | [Olimex STM32-E407](https://www.olimex.com/Products/ARM/ST/STM32-E407/open-source-hardware) | `zephyr olimex-stm32-e407` | 
+| [Zephyr](https://www.zephyrproject.org/) | [ST B-L475E-IOT01A](https://docs.zephyrproject.org/latest/boards/arm/disco_l475_iot1/doc/index.html) | `zephyr st-b-l475e-iot01a` | 
```

Comment:
Trying this out, I found that Zephyr networking support is not included, yet. Please mention this prominently.

Reply:
Zephyr seems to support networking out of the box, but it is requiered to check if they are POSIX network compliant and check wheter we can use the POSIX transports of Micro-XRCE-DDS of a custom one in needed. 

This is on our TODO list.

Reply:
Looks like they have a slightly different naming, but otherwise compatible: https://docs.zephyrproject.org/1.11.0/api/networking.html#bsd-sockets-compatible-api

Why they are not using exactly the same name in a "compatible" API ... no clue.

Reply:
Check this: https://docs.zephyrproject.org/latest/reference/networking/sockets.html

They have some BSD Socket POSIX API, let me check this in the following days.

Reply:
The reason that Zephyr uses a different naming can be found [here](https://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_NET_SOCKETS_POSIX_NAMES.html#cmdoption-arg-config-net-sockets-posix-names).

Reply:
It seems prudent not to enable that option, to prevent the potential issue with close.

Can you handle this in Micro-XRCE-DDS?